tests/python/gaia-ui-tests/gaiatest/apps/settings/regions/bluetooth.py

```diff
@@ -4,55 +4,61 @@
 
 from marionette_driver import expected, By, Wait
 
-from gaiatest.apps.base import PageRegion
+from gaiatest.apps.base import Base
```

Comment:
Ah, because of the stale element exception.  because I have to switch context for screen capture, I will run into this quite often.